packages/gestalt/src/Button.css

```diff
@@ -42,12 +42,24 @@
 }
 
 .smVr {
+  border-radius: var(--rounding-200);
+  min-height: 24px;
+  padding: var(--sema-space-100) var(--sema-space-200);
+}
+
+.smLinkVr {
   border-radius: var(--rounding-200);
   min-height: 28px;
   padding: var(--sema-space-100) var(--sema-space-200);
 }
 
 .mdVr {
+  border-radius: var(--rounding-300);
```

Comment:
can we use VR tokens when the classname is only VR?

.mdVr {
  border-radius: var(--sema-rounding-300);